### PR TITLE
Add Uniqlo

### PR DESCRIPTION
**Pull request by DanneelsSophie**
uniqlo

**Issue:** contributes to #9222 

**Similarweb rank:**
[https://www.similarweb.com/website/uniqlo.com/#overview
](https://www.similarweb.com/website/uniqlo.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by DanneelsSophie** [comment]
Is available for the hacktoberfest ? :D

**Comment by DanneelsSophie** [comment]
Is valid for the hacktoberfest ?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13982,6 +13982,11 @@
             "hex": "1F36C7",
             "source": "https://www.unilever.com/our-company/the-logo"
         },
+        {
+            "title": "Uniqlo",
+            "hex": "ED1D24",
```
Comment (adamrusted) [inline _data/simple-icons.json:13987]:
```suggestion
            "hex": "FF0000",
```

Their website SVG is in primary red - we should use that too.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @DanneelsSophie!
Unfortunately, I can see [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/7acb20b4-d0c5-45bd-a332-12eccb61f3ee) when comparing your icon in black to the source icon in red. Would you mind taking another look at this one?

File: _data/simple-icons.json (hunk 1 above)
Comment (DanneelsSophie) [reply _data/simple-icons.json:13987]: https://brandpalettes.com/uniqlo-colors/ I look this website

**Comment by PeterShaggyNoble** [comment]
Are you still working on fixing the differences Adam pointed out [above](https://github.com/simple-icons/simple-icons/pull/9754#pullrequestreview-1738486875), @DanneelsSophie?

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]

File: icons/uniqlo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Uniqlo</title><path d="M0 0v24h24V0Zm10.115 3.274h1.531l1.985 4.337V3.274h1.534v7.581h-1.534l-1.985-4.344v4.344h-1.53zm7.554 0h1.536v7.582H17.67zm-14.62 0h1.498v5.063c0 .575.456 1.038 1.022 1.038.568 0 1.02-.463 1.02-1.038V3.274h1.476v5.063c0 1.422-1.082 2.52-2.497 2.52a2.488 2.488 0 0 1-2.52-2.52zm7.066 9.879h1.531v6.037h3.52v1.542h-5.05Zm-4.548 0a2.485 2.485 0 0 1 2.498 2.504v2.57c0 .655-.29 1.293-.7 1.762l.7 2.278H6.589l-.527-1.583a2.519 2.519 0 0 1-3.015-2.457v-2.57a2.507 2.507 0 0 1 2.52-2.504zm12.868 0a2.488 2.488 0 0 1 2.497 2.504v2.57a2.499 2.499 0 1 1-4.996 0v-2.57a2.487 2.487 0 0 1 2.499-2.504zM5.567 14.636a1.017 1.02 0 0 0-1.02 1.021v2.57c0 .499.455.963 1.02.963.57 0 1.022-.464 1.022-.962v-2.57c0-.567-.452-1.022-1.022-1.022zm12.868 0c-.565 0-1.025.455-1.025 1.021v2.57c0 .499.46.963 1.025.963.562 0 1.024-.464 1.024-.962v-2.57c0-.567-.462-1.022-1.024-1.022z"/></svg>
```
[the remark this answers is not in the document]
Comment (DanneelsSophie) [reply icons/uniqlo.svg]: Sorry for the delay, thks updating my pr :)

[the remark this answers is not in the document]
Comment (PeterShaggyNoble) [reply icons/uniqlo.svg]: No problem, @DanneelsSophie. Sorry, for _usurping_ your PR; we're on a bit of a rampage at the moment trying to close out as many issues & PRS as we can, to get numbers down to a more manageable level. Thanks for you initial work on this, though 🙂

**Pull request merged**